src/main/kotlin/com/github/michaelbull/retry/policy/Backoff.kt

```diff
@@ -52,7 +52,7 @@ fun fullJitterBackoff(base: Long, max: Long): RetryPolicy<*> {
 
         /* sleep = random_between(0, min(cap, base * 2 ** attempt)) */
         val delay = min(max, base saturatedMultiply attempt.binaryExponential())
-        val randomDelay = random.nextLong(delay saturatedAdd 1)
+        val randomDelay = random.nextLong(1L,delay saturatedAdd 1)
 
         RetryAfter(randomDelay)
```

Comment:
The solution here should be to let it still roll a 0, but if it does return an instruction of `RetryImmediately` rather than `RetryAfter`